src/components/IssueUI/BTCPaymentPendingStatusUI/index.tsx

```diff
@@ -70,7 +75,7 @@ const BTCPaymentPendingStatusUI = ({ request }: Props): JSX.Element => {
             'block'
           )}
         >
-          {`≈ $ ${getUsdAmount(amountBTCToSend, prices.bitcoin?.usd)}`}
+          {`≈ $ ${getUsdAmount(amountBTCToSend, getTokenPrice(prices, 'BTC')?.usd)}`}
```

Comment:
Thoughts on accessing USD property like this? Doesn't take us very far from what we were doing before, but arguably would be neater to pass in a currency parameter:

`getTokenPrice(prices, 'BTC', 'usd')` or similar.

Think I'd prefer to leave it as it is though, unless anyone has any strong opinions on it.